src/sentry/static/sentry/app/components/queryCount.tsx

```diff
@@ -37,9 +39,9 @@ const QueryCount = ({
 
   return (
     <div className={cx}>
-      <span>(</span>
+      {!hideParens && <span>(</span>}
```

Comment:
Are we sure we want to do this?
When you swap to a tab and then do a custom search, the last tab you were on still has the count, which looks inconsistent.
I would prefer to have the counts visible always.